### PR TITLE
Feature/added header based logger + added`risk_start_time` and `enforcer_start_time` fields to activities 

### PR DESCRIPTION
**Pull request by chen-zimmer-px**

### CODE REVIEW

**Review by ori-gold-px — CHANGES_REQUESTED** [review]
Looks simple enough. Just two things:

(1) holding off on approving until node core is updated and released so package.json can be updated
(2) E2E tests are failing - please resolve

File: px_metadata.json [hunk 1]
```diff
@@ -58,6 +59,14 @@
         "test_page_requested_activity_cookie_origin",
         "test_block_page_hard_block_response",
         "test_risk_api_validate_cookie_origin",
-        "test_risk_cookie_valid_cookie_with_user_agent_bigger_than_max_length"
+        "test_risk_cookie_valid_cookie_with_user_agent_bigger_than_max_length",
+        "test_risk_api_schema",
+        "test_block_activity_schema",
+        "test_page_requested_activity_schema",
```
Comment (ori-gold-px) [inline px_metadata.json:65]: You know you can simplify these three lines with this:

```
"test_(risk_api|block_activity|page_requested_activity)_schema"
```

File: px_metadata.json [hunk 2]
```diff
@@ -58,6 +59,14 @@
...
+        "test_risk_cookie_validate_client_uuid_in_risk_api",
+        "test_block_activity_headers",
+        "test_page_requested_activity_headers",
```
Comment (ori-gold-px) [inline px_metadata.json:68]: Same here:

```
"test_(block|page_requested)_activity_headers"
```

File: px_metadata.json [hunk 3]
```diff
@@ -58,6 +59,14 @@
...
+        "test_sensitive_headers_removed_from_page_requested",
+        "test_sensitive_headers_removed_from_block"
```
Comment (ori-gold-px) [inline px_metadata.json:70]:
```
"test_sensitive_headers_removed_from_(block|page_requestd)"
```

**Review by ori-gold-px — APPROVED** [review]

File: px_metadata.json (hunk 1 above)
Comment (chen-zimmer-px) [reply px_metadata.json]: changed

File: px_metadata.json (hunk 3 above)
Comment (chen-zimmer-px) [reply px_metadata.json]: changed

File: px_metadata.json (hunk 2 above)
Comment (chen-zimmer-px) [reply px_metadata.json]: changed

**Pull request merged**